src/components/Dashboard/styles.postcss

```diff
@@ -80,47 +80,57 @@
 }
 
 .btn-dashboard-switcher {
-  display: none;
+  flex: 1 1 100%;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: rem(35);
+  height: rem(35);
```

Comment:
Little big detail, this button doesn't look like at the design. Please, take a look at zeplin.